Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionCellBuilder.swift

```diff
@@ -55,11 +55,13 @@ class ReviewExecutionCellBuilder: TransactionDetailCellBuilder {
         paymentGroupCell.tableView.registerCell(DisclosureWithContentCell.self)
         paymentGroupCell.tableView.registerCell(SecondaryDetailDisclosureCell.self)
         paymentGroupCell.tableView.registerCell(PaymentMethodCell.self)
-
+        paymentGroupCell.tableView.separatorStyle = .none
         let estimatedFeeCell = buildEstimatedGasFee(model.feeState, tableView: paymentGroupCell.tableView)
 
         if case let .filled(relayerInfo) = model.relayerState,
-           relayerInfo.remainingRelays > ReviewExecutionViewController.MIN_RELAY_TXS_LEFT && !userSelectedSigner {
+           relayerInfo.remainingRelays > ReviewExecutionViewController.MIN_RELAY_TXS_LEFT &&
+            !userSelectedSigner &&
+            safe.chain!.isSupport(feature: .relay) {
```

Comment:
relayerInfo should not be .filled at this point. No need to check if relayer feature is supported here. It should have been checked before.